### ISSUE TITLE
Updated Training Set and New Results -- L3C v2

### ISSUE THREAD

**Issue by fab-jul** [fab-jul/L3C-PyTorch#14]
Previously, our preprocessing script saved all training images and validation images as JPGs with a high quality factor of Q=95, downscaled by a factor 0.75. It turns out that the resulting images have a specific enough distribution that the neural network picks up on it, and the images are also easier to compress for the non-learned codecs. 

For correctness, we have thus re-created the training and validation sets. The new preprocessing script is available in the repo. The important differences are:
- All images are saved as PNGs.
- We do not rescale validation sets in any way, and instead divide the images into crops such that everything fits into memory. Note that this is a bias _against_ our method, since more context can only help. We only crop images too big to fit into our GPU (TITAN X Pascal). Please see the updated README.
- For the training set, we use a random downscaling factor, instead of fixed 0.75x: this provides a wider variety of downscaling artefacts.
- Additionally, we use the Lanczos filter, as we found that Bicubic also introduces specific artefacts.

This causes all results to shift, however, as before, we still **outperform WebP, JPEG-2000, and PNG**, i.e. the ordering of the methods according to bpp remains unchanged.

We evaluated our model on 500 images randomly selected from the Open Images validation set, and preprocessed like the 
training data: [Download Open Images 500](http://data.vision.ee.ethz.ch/mentzerf/validation_sets_lossless/val_oi_500_r.tar.gz)

## Updated ArXiv

We updated the arxiv version of the paper, which will be online soon.

## New Results

&nbsp;  | Method | Open Images | DIV2K | RAISE-1k
--- | --- | --- | --- | ---
Ours | L3C v2 | 2.990597132 | 3.093768752 | 2.386501087
Learned Baselines | RGB Shared | 4.313588005 | 4.429001861 | 3.779201962
&nbsp; | RGB | 3.297824781 | 3.418117799 | 2.572320659 |  | 3.246318873
Non-Learned Approaches | PNG | 4.004512908 | 4.234729262 | 3.556403138
&nbsp; | JPEG2000  | 3.054759549 | 3.126744435 | 2.46459739
&nbsp; | WebP | 3.047477818 | 3.176081706 | 2.461481317
&nbsp; | FLIF | 2.866778476 | 2.910950783 | 2.084036243

## Status

This is currently in feat/v3, and will be merged soon.

**Comment by fab-jul** [comment]
Arxiv is now available [https://arxiv.org/abs/1811.12817v3](https://arxiv.org/abs/1811.12817v3).

Merged feat/v3 into master. 

Please let me know here if there are issues with this release.

**Comment by DonaldTsang** [comment]
Sorry for cross-posting but https://github.com/caoscott/SReC/issues/6 has addressed that there are newer image formats like AVIF, HEIF and JPEG-XL (JXL).
